metricbeat/module/system/process_summary/_meta/data.json

```diff
@@ -15,14 +15,10 @@
     "system": {
         "process": {
             "summary": {
-                "dead": 0,
-                "idle": 63,
-                "running": 3,
-                "sleeping": 117,
-                "stopped": 0,
-                "total": 185,
-                "unknown": 2,
-                "zombie": 0
+                "idle": 122,
+                "running": 1,
+                "sleeping": 222,
+                "total": 345
```

Comment:
Why did the list of process states get smaller here?

Reply:
It no longer reports states where no processes on the system are in that state. Kind of on the fence with this change, but I figured it was less confusing than having a list of highly OS/kernel version specific process states where only one or two are reported to the user.

Reply:
Ah that makes sense to me, thanks.